extensions/amp-ad/amp-ad.md

```diff
@@ -1,6 +1,8 @@
 # <a name="amp-ad"></a> `amp-ad` / `amp-embed`
 
-NOTE: The specification of `amp-ad` / `amp-embed` is likely to significantly evolve over time. The current approach is designed to bootstrap the format to be able to show ads.
+{% call callout('Note', type='note') %}
```

Comment:
Yes, all callouts in GitHub don't look nice.  I'll make a note about automating callouts in my list of things to do.